bokeh/properties.py

```diff
@@ -175,7 +175,8 @@ def matches(self, new, old):
         except (KeyboardInterrupt, SystemExit):
             raise
         except Exception as e:
-            logger.debug("could not compare %s and %s for property %s (Reason: %s)", new, old, self.name, e)
+            # if we cannot compare (e.g. arrays) just punt return False for match
+            pass
```

Comment:
I'm not sure. I was thinking about it, if two things aren't comparable, matching false seems reasonable, and if there is a problem with the types, that should be could by the properties type checker. I think this is actually ok?